test/Microsoft.AspNet.Razor.Test/Generator/CSharpTagHelperRenderingTest.cs

```diff
@@ -59,50 +30,64 @@ public static TheoryData TagHelperDescriptorFlowTestData
                     {
                         "SingleTagHelper",
                         "SingleTagHelper",
-                        PAndInputTagHelperDescriptors,
-                        PAndInputTagHelperDescriptors,
+                        DefaultPAndInputTagHelperDescriptors,
+                        DefaultPAndInputTagHelperDescriptors,
                         false
                     },
                     {
                         "SingleTagHelper",
                         "SingleTagHelper.DesignTime",
-                        PAndInputTagHelperDescriptors,
-                        PAndInputTagHelperDescriptors,
+                        DefaultPAndInputTagHelperDescriptors,
+                        DefaultPAndInputTagHelperDescriptors,
                         true
                     },
                     {
                         "BasicTagHelpers",
                         "BasicTagHelpers",
-                        PAndInputTagHelperDescriptors,
-                        PAndInputTagHelperDescriptors,
+                        DefaultPAndInputTagHelperDescriptors,
+                        DefaultPAndInputTagHelperDescriptors,
                         false
                     },
                     {
                         "BasicTagHelpers",
                         "BasicTagHelpers.DesignTime",
-                        PAndInputTagHelperDescriptors,
-                        PAndInputTagHelperDescriptors,
+                        DefaultPAndInputTagHelperDescriptors,
+                        DefaultPAndInputTagHelperDescriptors,
                         true
                     },
                     {
                         "BasicTagHelpers.RemoveTagHelper",
                         "BasicTagHelpers.RemoveTagHelper",
-                        PAndInputTagHelperDescriptors,
+                        DefaultPAndInputTagHelperDescriptors,
                         Enumerable.Empty<TagHelperDescriptor>(),
                         false
                     },
                     {
+                        "BasicTagHelpers.Prefixed",
+                        "BasicTagHelpers.Prefixed",
+                        BuildPAndInputTagHelperDescriptors(prefix: "THS"),
+                        BuildPAndInputTagHelperDescriptors(prefix: "THS"),
```

Comment:
nit: create a `static` `PrefixedPAndInputTagHelperDescriptors` property right next to `DefaultPAndInputTagHelperDescriptors`